cmd/homeassistant_options.go

```diff
@@ -43,14 +43,19 @@ var homeassistantOptionsCmd = &cobra.Command{
 			options["startup_time"] = startupTime
 		}
 
+		waitBoot, err := cmd.Flags().GetInt("wait_boot")
```

Comment:
That is a supervisor option

Reply:
as agreed in the chat, this is the correct option,